envisage/extension_point_binding.py

```diff
@@ -23,15 +23,15 @@ class ExtensionPointBinding(HasTraits):
     #### 'ExtensionPointBinding' interface ####################################
 
     # The object that we are binding the extension point to.
-    obj = Any
+    obj = WeakRef
 
     # The Id of the extension point.
     extension_point_id = Str
 
     # The extension registry used by the binding. If this trait is not set then
     # the class-scope extension registry set on the 'ExtensionPoint' class is
     # used (and if that is not set then the binding won't work ;^)
-    extension_registry = Instance(IExtensionRegistry)
+    extension_registry = WeakRef(IExtensionRegistry)
```

Comment:
Is this change necessary to fix #97? If all the target objects are already garbage collected, won't the extension registry also have been collected?